src/main/java/core/basesyntax/strategy/SupplyOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
+    private final Storage storage;

```

Comment:
According to the encapsulation principle, the `storage` field should be private if it's not intended to be exposed or used by other classes directly.